src/virtual_machine/lc3_virtual_machine.rs

```diff
@@ -0,0 +1,384 @@
+use std::io::Read;
+
+use super::{instructions::*, register::Register, trap::Trap};
+
+const AMMOUNT_OF_REGISTERS: usize = 10;
```

Comment:
I think this is a bit weird. First off, it probably belongs in the `register` module. Furthermore, there already is a struct with the known registers. Having a second "source of truth" about the registers is unexpected.

Reply:
That's true, the creation of the `register` module was a last-second change and missed that one